src/test/proposals/49.sol

```diff
@@ -0,0 +1,70 @@
+pragma solidity ^0.6.7;
+pragma experimental ABIEncoderV2;
+
+import "../SimulateProposalBase.t.sol";
+import "../../GebDaoGovernanceActions.sol";
+
+contract Proposal49Test is SimulateProposalBase {
+    function test_proposal_49() public onlyFork {
+        PIRawPerSecondCalculatorLike controller = PIRawPerSecondCalculatorLike(
+            0x5CC4878eA3E6323FdA34b3D28551E1543DEe54C6
+        ); // GEB_RRFM_CALCULATOR
+
+        address[] memory targets = new address[](3);
+        bytes[] memory calldatas = new bytes[](3);
+
+        // RAI Controller = https://etherscan.io/address/0x5cc4878ea3e6323fda34b3d28551e1543dee54c6
+        // HAI Controller = https://optimistic.etherscan.io/address/0x6f9aeC3c0DF4DF7A0Da66453a38B8C767972f609
+
+        // Current RAI values
+        // KP = sg = 222002205862
+        // KI = ag = 16442
+        // decay = pscl = 999999711200000000000000000
+
+        // Current RAI values
+        // KI = ag = 222002205862
+        // KP = sg = 16442
+        // decay = pscl = 999999711200000000000000000
+
```

Comment:
```suggestion
```
This is a duplicate comment, beware the second one contains wrong values, need to keep the first one.

Reply:
good catch, thank you